src/api/spec/models/user_spec.rb

```diff
@@ -29,37 +29,42 @@
     end
   end
 
-  describe '#update_notifications' do
-    context 'when valid' do
-      it 'updates one User notification' do
-        admin_user.update_notifications(input)
+  shared_examples 'updates notifications' do
+    before do
+      # If the parameter is the User class, the user_id has to be nil
+      @id = object.instance_of?(User) ? object.id : nil
+    end
+
+    context 'when valid'do
+      it 'updates one global notification' do
+        object.update_notifications(input)
         expect(
-          EventSubscription.exists?(user_id: admin_user, eventtype: 'Event::RequestCreate', receiver_role: 'source_maintainer', receive: true)
+          EventSubscription.exists?(user_id: @id, eventtype: 'Event::RequestCreate', receiver_role: 'source_maintainer', receive: true)
```

Comment:
because it's set in the before block I assume it has to be an instance variable, however, I haven't tried it. I can investigate, will keep you updated....